docs/md/python.md

```diff
@@ -481,6 +481,10 @@ PerspectiveWidget(table)
 PerspectiveWidget(None)
 ```
 
+## `PerspectiveRenderer`
+
+Perspective also exposes a JS-only `mimerender-extension`. This lets you view `csv`, `json`, and `arrow` files directly from the file browser. You can see this by right clicking one of these files and `Open With->CSVPerspective` (or `JSONPerspective` or `ArrowPerspective`). Perspetive will also install itself as the default handler for opening `.arrow` files.
```

Comment:
Perspetive

Reply:
Also is there a call-to-action here?  If this is just an ad for the functionality, maybe roll the paragraph into the `PerspectiveWidget` adjacent sibling?

Reply:
Mostly an ad, but its different from `PerspectiveWidget` since `PerspectiveWidget` is available in certain contexts (JupyterLab, Voilà, soon nteract and legacy Jupyter) where the Perspective Renderer is not (only available in JupyterLab).